src/components/MainPage.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable object-curly-newline */
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable no-unused-vars */
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
+import { Link } from 'react-router-dom';
+
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { getApiDataTour } from '../redux/tours/tours';
+import { fetchApiDataTours } from '../redux/tours/toursAPI';
+import '../styles/main.css';
+import Dots from './Dots';
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+
+const MainPage = () => {
+  const user = useSelector((store) => store.user);
+  const tours = useSelector((store) => store.tours);
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    dispatch(fetchApiDataTours());
+  }, [dispatch]);
+
+  return (
+    <main className="main-page">
+      <div className="header">
+        <h1>OUR LATEST TOURS</h1>
+        <p>
+          {user.length > 0 ? `Hello ${user[0].user.name}, ` : ''}
+          Please select a Tour
+        </p>
+        <Dots />
+      </div>
+
+      <Swiper
```

Comment:
- [ ] OPTIONAL: This is just nitpicking on your UX.  The swiper arrows are inside the tour link next to them:

![image](https://user-images.githubusercontent.com/57421823/199877313-e232a4c9-4398-437a-a5b9-b6956e79cb66.png)

Therefore, if I click the arrow more than once (which everybody does), I end up inside the tour, instead of at the end of the carousel, as one would expect.

I would make the link smaller so the arrow is free for all your users' clicking enjoyment when they reach the end of the carousel.

I was caught in this "trap" like 20 times ending up inside the last (or the first) tour details, instead of just hitting the end of the carousel. 

Of course, this is just a suggestion to improve the UX (user experience and usability).

Reply:
@Meltrust Sorry for the inconvenience. We will fix that soon. Thank you so much for your kind suggestions.

I would love to connect with you.

👤 **ABDUL ALI**

> * This is My **GitHub**: ![GitHub followers](https://img.shields.io/github/followers/ABDULALI3468?label=ABDULALI&style=social)
> * This is My **Linkedin**: [![Linkedin Badge](https://img.shields.io/badge/-ABDUL%20ALI-blue?style=flat-square&logo=Linkedin&logoColor=white&link=https://www.linkedin.com/in/abdul-ali-5400bb216/)](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [![Twitter Badge](https://img.shields.io/badge/-@mrabdul_ali_-1ca0f1?style=flat-square&labelColor=1ca0f1&logo=twitter&logoColor=white&link=https://twitter.com/mrabdul_ali)](https://twitter.com/mrabdul_ali)&nbsp;